src/Riverline/MultiPartParser/StreamedPart.php

```diff
@@ -36,6 +36,13 @@ class StreamedPart
      */
     private $parts = array();
 
+    /**
+     * The length of the EOL character.
+     *
+     * @var int
+     */
+    private static $EOLCharacterLength = 2;
```

Comment:
not really happy with the static variable.

Why not have a object variable instead?

Reply:
Any chance to have tests for this?